src/core/Slider/Slider.test.tsx

```diff
@@ -0,0 +1,934 @@
+  expect(thumb).toBeFalsy();
+});
+
+it('should render disabled component', () => {
+  const { container } = render(<Slider values={defaultSingleValue} disabled />);
+  assertBaseElement(container);
+  expect(
+    container.querySelector('.iui-slider-component-container.iui-disabled'),
+  ).toBeTruthy();
+  const thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeTruthy();
+  expect(thumb.getAttribute('aria-disabled')).toEqual('true');
+});
+it('should render min max labels by default', () => {
+  const { container } = render(<Slider values={defaultSingleValue} />);
+  assertBaseElement(container);
+  expect(
+    container.querySelector('.iui-slider-component-container'),
+  ).toBeTruthy();
+  expect(
+    (container.querySelector('.iui-slider-min') as HTMLSpanElement).textContent,
+  ).toBe('0');
+  expect(
+    (container.querySelector('.iui-slider-max') as HTMLSpanElement).textContent,
+  ).toBe('100');
+});
+it('should render specified min max labels', () => {
+  const { container } = render(
+    <Slider values={defaultSingleValue} min={5} max={55} />,
+  );
+  assertBaseElement(container);
+  expect(
+    container.querySelector('.iui-slider-component-container'),
+  ).toBeTruthy();
+  expect(
+    (container.querySelector('.iui-slider-min') as HTMLSpanElement).textContent,
+  ).toBe('5');
+  expect(
+    (container.querySelector('.iui-slider-max') as HTMLSpanElement).textContent,
+  ).toBe('55');
+});
+it('should render provided min max labels', () => {
+  const { container } = render(
+    <Slider values={defaultSingleValue} maxLabel='big' minLabel='small' />,
+  );
+  assertBaseElement(container);
+  expect(
+    container.querySelector('.iui-slider-component-container'),
+  ).toBeTruthy();
+  expect(
+    (container.querySelector('.iui-slider-min') as HTMLSpanElement).textContent,
+  ).toBe('small');
+  expect(
+    (container.querySelector('.iui-slider-max') as HTMLSpanElement).textContent,
+  ).toBe('big');
+});
+it('should render provided min max label nodes', () => {
+  const { container } = render(
+    <Slider
+      values={defaultSingleValue}
+      maxLabel={<span className='span-max'>big</span>}
+      minLabel={<span className='span-min'>small</span>}
+    />,
+  );
+  assertBaseElement(container);
+  expect(
+    container.querySelector('.iui-slider-component-container'),
+  ).toBeTruthy();
+  expect(
+    (container.querySelector('.span-min') as HTMLSpanElement).textContent,
+  ).toBe('small');
+  expect(
+    (container.querySelector('.span-max') as HTMLSpanElement).textContent,
+  ).toBe('big');
+});
+it('should render provided min max label nodes', () => {
+  const { container } = render(
+    <Slider
+      values={defaultSingleValue}
+      maxLabel={<span className='span-max'>big</span>}
+      minLabel={<span className='span-min'>small</span>}
+    />,
+  );
+  assertBaseElement(container);
+  expect(
```

Comment:
amm.. assertBaseElement checks the container already.. I suggest either add more checks in that method (rail, track, etc) or remove that function.